app/controllers/CAS.scala

```diff
@@ -0,0 +1,24 @@
+package controllers
+
+import actions.CommonActions._
+import com.gu.subscriptions.CAS.{CASError, CASSuccess}
+import configuration.Config
+import forms.CASForm
+import play.api.libs.json.Json
+import play.api.mvc.Controller
+import views.support.CASResultOps._
+import play.api.libs.concurrent.Execution.Implicits._
+
+object CAS extends Controller {
+  def index = GoogleAuthenticatedStaffAction { implicit request =>
+    Ok(views.html.staff.cas())
+  }
+
+  def search = GoogleAuthenticatedStaffAction.async(parse.form(CASForm())) { request =>
```

Comment:
`GoogleAuthenticatedStaffAction` is good - tho' I'm surprised to notice that, [unlike Membership](https://github.com/guardian/membership-frontend/blob/1fe86747a6/frontend/app/controllers/Testing.scala#L18-L28), we don't have any code in place to restrict access based on Google-Group membership - ideally, we should really be restricting use of this tool to `userhelp@` and ourselves.